Php unit tests are failing in nightly CI

Php unit tests are failing in following piplines
1. phpunit-scality-mbuck-cov-php7.4-10.12.0-sqlite
2. phpunit-ceph-cov-php7.4-10.12.0-sqlite
3. phpunit-ceph-cov-php7.4-10.13.0-sqlite

See build for detailed report ; https://drone.owncloud.com/owncloud/files_primary_s3/3879/8/8
```console
127) OCA\Files_Versions\Tests\VersioningTest::testStoreVersionAsRecipient with data set "metaEnabled" (true)

/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1416
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1365
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1329
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:97

128) OCA\Files_Versions\Tests\VersioningTest::testStoreVersionAsAnonymous with data set "metaDisabled" (false)

/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1416
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1365
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1356
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:97

129) OCA\Files_Versions\Tests\VersioningTest::testStoreVersionAsAnonymous with data set "metaEnabled" (true)

/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1416
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1365
/var/www/owncloud/server/apps/files_versions/tests/VersioningTest.php:1356
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:97

OK, but incomplete, skipped, or risky tests!
Tests: 10724, Assertions: 59434, Skipped: 129.

make: *** [Makefile:98: test-php-unit-dbg] Killed
Generating code coverage report in Clover XML format ... 
```

--- a/.drone.star
+++ b/.drone.star
@@ -76,8 +76,7 @@ config = {
             ],
             "servers": [
                 "daily-master-qa",
-                "10.12.0-qa",
-                "10.13.0-qa",
+                "10.13.4-qa",
             ],
             "databases": [
                 "sqlite",
@@ -102,8 +101,7 @@ config = {
             ],
             "servers": [
                 "daily-master-qa",
-                "10.12.0-qa",
-                "10.13.0-qa",
+                "10.13.4-qa",
             ],
             "scalityS3": {
                 "config": "multibucket",
@@ -126,8 +124,7 @@ config = {
             ],
             "servers": [
                 "daily-master-qa",
-                "10.12.0-qa",
-                "10.13.0-qa",
+                "10.13.4-qa",
             ],
             "cephS3": True,
             "includeKeyInMatrixName": True,

